components/analytics/compact-dashboard.tsx

```
@@ -0,0 +1,781 @@
+                                                                key={`cell-${idx}`}
+                                                                fill={slice.color}
+                                                                style={{ cursor: 'pointer' }}
+                                                                stroke="#1e1e22"
+                                                                strokeWidth={1}
+                                                            />
+                                                        ))}
+                                                    </Pie>
+                                                    <Tooltip
+                                                        formatter={(value, name) => {
+                                                            const numValue = Number(value);
+                                                            if (!isNaN(numValue)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)